### PR TITLE
Name change

### PR DESCRIPTION
**Pull request by Themichaelreimer**
averagePlayerDistanceFromSpawn -> PlayerIDSFromSpawn;
PlayerIDSFromSpawn -> playerIDSFromSpawn

### CODE REVIEW

File: lua/cfc_random_spawn/module/sv_player_spawning.lua [hunk 1]
```diff
@@ -36,31 +36,35 @@ if mapHasCustomSpawns then
         return cfcRandomSpawn.spawnPointRankings[randomSpawn]["spawn"]
     end
 
-    function cfcRandomSpawn.updateSpawnPointRankings()
-        local averagePlayerDistanceFromSpawns = {}
+    function cfcRandomSpawn.updateSpawnPointRankings( ply )
+        local playerIDSFromSpawns = {}
```
Comment (sarahsturgeon) [inline lua/cfc_random_spawn/module/sv_player_spawning.lua:40]: Spacing is off, and it should be singular

File: lua/cfc_random_spawn/module/sv_player_spawning.lua [hunk 2]
```diff
@@ -36,31 +36,35 @@ if mapHasCustomSpawns then
...
-    function cfcRandomSpawn.updateSpawnPointRankings()
-        local averagePlayerDistanceFromSpawns = {}
+    function cfcRandomSpawn.updateSpawnPointRankings( ply )
+        local playerIDSFromSpawns = {}
+	local playerPVPStatus = ply:GetNWBool("CFC_PvP")
 
         for _, spawn in pairs( customSpawnsForMap ) do
-            local playerNetForce = getPlayerForceFromCustomSpawn( spawn )
+			
+	    if playerPVPStatus == spawn["pvp"]
```
Comment (sarahsturgeon) [inline lua/cfc_random_spawn/module/sv_player_spawning.lua:45]: I don't think this belongs in this PR?

File: lua/cfc_random_spawn/module/sv_player_spawning.lua [hunk 3]
```diff
@@ -36,31 +36,35 @@ if mapHasCustomSpawns then
...
 
     --timer.Create( "CFC_UpdateOptimalSpawnPosition", 0.5, 0, cfcRandomSpawn.updateSpawnPointRankings )
 
     function cfcRandomSpawn.handlePlayerSpawn( ply )
-        if not (ply && IsValid( ply )) then return end
+        if not ( ply && IsValid( ply ) ) then return end
         if ply.LinkedSpawnPoint then return end
 		
-		cfcRandomSpawn.updateSpawnPointRankings()
+	cfcRandomSpawn.updateSpawnPointRankings( ply )
```
Comment (sarahsturgeon) [inline lua/cfc_random_spawn/module/sv_player_spawning.lua:67]: Looks like spacing is still off by 1 here

File: lua/cfc_random_spawn/module/sv_player_spawning.lua [hunk 4]
```diff
@@ -36,31 +36,35 @@ if mapHasCustomSpawns then
...
-        cfcRandomSpawn.spawnPointRankings = PlayerIDSFromSpawn
+        cfcRandomSpawn.spawnPointRankings = playerIDSFromSpawn
         table.SortByMember( cfcRandomSpawn.spawnPointRankings, "inverse-distance-squared", true )
     end
 
     --timer.Create( "CFC_UpdateOptimalSpawnPosition", 0.5, 0, cfcRandomSpawn.updateSpawnPointRankings )
 
     function cfcRandomSpawn.handlePlayerSpawn( ply )
-        if not (ply && IsValid( ply )) then return end
+        if not ( ply && IsValid( ply ) ) then return end
```
Comment (sarahsturgeon) [inline lua/cfc_random_spawn/module/sv_player_spawning.lua:64]: 👍

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]
Small stuff

File: lua/cfc_random_spawn/module/sv_player_spawning.lua (hunk 2 above)
Comment (Themichaelreimer) [reply lua/cfc_random_spawn/module/sv_player_spawning.lua:45]: No? The idea is to only consider the set of spawn points which match our pvp status, by skipping over entries tagged with a different pvp status

**Pull request closed without merging**